source/documentation/deploying_services/use_a_custom_domain.erb

```diff
@@ -360,12 +360,27 @@ Custom domains are configured by our cdn-route service which uses a CloudFront d
 
 ### Caching
 
-CloudFront uses your app's `Cache-Control` or `Expires` HTTP headers to determine [how long to cache content](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Expiration.html). If your app does not provide these headers, CloudFront will use a default timeout of 24 hours. This can be particularly confusing as different requests might be routed to different CloudFront Edge endpoints.
+CloudFront uses your app's `Cache-Control` or `Expires` HTTP headers to determine [how long to cache content](https://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Expiration.html).
+
+If your app does not provide these headers, CloudFront will use the configured default time-to-live. This can be particularly confusing as different requests might be routed to different CloudFront Edge endpoints.
+
+The default time-to-live is set upon creation and is:
```

Comment:
What is created and what does the creating? When you create your CDN distribution?

If so, suggest: "When you create your CDN distribution, CloudFront sets the default time-to-live at:"

Reply:
Resolvedi in 25a632bebe4593bd3b7b76ea5fc2ad98fb7d195a